scripts/Tachidesk-Server-x64.wxs

```diff
@@ -0,0 +1,86 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi">
+  <Product Id="*" UpgradeCode="*"
+    Version="0.6.0" Language="1033" Name="Tachidesk Server" Manufacturer="Suwayomi">
+    <Package InstallerVersion="300" Compressed="yes" />
+    <Media Id="1" Cabinet="Tachidesk_Server.cab" EmbedCab="yes" />
+
+    <Condition Message="This version of Windows does not support deploying 64-bit packages.">
+      VersionNT64
+    </Condition>
+
+    <!-- Directory -->
+    <Directory Id="TARGETDIR" Name="SourceDir">
+      <Directory Id="ProgramFiles64Folder">
+        <Directory Id="INSTALLDIR" Name="Tachidesk-Server" >
+          <Directory Id="jre"/>
+          <Directory Id="electron"/>
+        </Directory>
+      </Directory>
+      <Directory Id="ProgramMenuFolder">
+        <Directory Id="ProgramMenuDir" Name="Tachidesk-Server">
+          <Component Id="ProgramMenuDir" Guid="*">
+            <RemoveFolder Id="ProgramMenuDir" On="uninstall"/>
+            <RegistryValue Root="HKCU" Key="Software\[Manufacturer]\[ProductName]" Type="string" Value="" KeyPath="yes"/>
```

Comment:
Is this correct? We don't use registry keys afaik?

Reply:
It's necessary for a clean uninstall

docs: 
https://wixtoolset.org/documentation/manual/v3/howtos/files_and_registry/create_start_menu_shortcut.html
https://www.firegiant.com/wix/tutorial/getting-started/the-files-inside/